ajax/libs/jquery-tokeninput/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "jquery-tokeninput",
+  "filename": "jquery.tokeninput.min.js",
+  "version": "1.7.0",
+  "description": "A jQuery Tokenizing Autocomplete Text Entry",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/loopj/jquery-tokeninput.git"
+  },
+  "keywords": [
+    "jquery",
+    "tokeninput"
+  ],
+  "author": "James Smith <james@loopj.com> (https://loopj.com)",
+  "homepage": "https://loopj.com/jquery-tokeninput/",
+  "license": "MIT",
```

Comment:
https://github.com/loopj/jquery-tokeninput/blob/master/bower.json#L21
hmmm... Looks like it's a dual license, the problem is how to confirm the GPL license version.